driver/compenv.ml

```diff
@@ -428,8 +426,6 @@ let read_one_param ppf position name v =
      let if_on = if name = "timings" then [ `Time ] else Profile.all_columns in
      profile_columns := if check_bool ppf name v then if_on else []
 
-  | "plugin" -> !load_plugin v
```

Comment:
My feeling is that where `-plugin` wants to remain in the CLI, leaving handling here could be annoying (on the basis that `OCAMLRUNPARAM` comes from your shell, not the specific invocation of the driver binary.

Arguably we could emit a warning instead? (doesn't have to be in this GPR)

Reply:
I think this is fine, at least for the moment.